src/pages/Movie/Cast/__tests__/component.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Cast from '../component';
+
+describe('Cast component', () => {
+  it('matches snapshot with required props', () => {
+    const requiredProps = {
+      cast: [{
+        name: 'Mock name',
+        character: 'Mock character'
+      }]
+    };
+    /* eslint-disable react/jsx-props-no-spreading */
```

Comment:
You can simplify this with next code and avoid line where you enable eslint:
```
/* eslint-disable-next-line react/jsx-props-no-spreading */
const wrapper = shallow(<Cast {...requiredProps} />);
```

Reply:
Thanks. Take note.